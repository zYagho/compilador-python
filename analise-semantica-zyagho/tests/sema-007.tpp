{Aviso: Variável 'a' já declarada anteriormente com o tipo 'inteiro'}
{Aviso: Variável 'b' declarada e inicializada, mas não utilizada}
{Aviso: Variável 'a' declarada e não inicializada}
{Erro: Função 'principal' deveria retornar 'inteiro', mas retorna 'vazio'}

inteiro: a
flutuante: a

inteiro principal()
	flutuante: a
	flutuante: b

	b := a + 1.0

fim
