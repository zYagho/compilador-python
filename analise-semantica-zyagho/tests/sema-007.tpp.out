WAR-SEM-VAR-DECL-PREV
WAR-SEM-VAR-DECL-INIT-NOT-USED
WAR-SEM-VAR-DECL-NOT-INIT
ERR-SEM-FUNC-RET-TYPE-ERROR
